lib/src/test/java/com/auth0/android/jwt/ClaimImplTest.java

```diff
@@ -241,4 +241,43 @@ public void shouldThrowIfListClassMismatch() throws Exception {
         exception.expect(DecodeException.class);
         claim.asList(UserPojo.class);
     }
+
+    @Test
+    public void shouldGetAsObject() throws Exception {
+        UserPojo data = new UserPojo("George", 1);
+        JsonElement value = gson.toJsonTree(data);
+        ClaimImpl claim = new ClaimImpl(value);
+
+        JsonElement value2 = gson.toJsonTree(1);
+        ClaimImpl claim2 = new ClaimImpl(value2);
+
+        JsonElement value3 = gson.toJsonTree(true);
+        ClaimImpl claim3 = new ClaimImpl(value3);
+
+        assertThat(claim.asObject(UserPojo.class), is(notNullValue()));
```

Comment:
Was not going to mention this but given that a test is already missing, please consider renaming these for better readability:
- claim -> userClaim
- claim2 -> intClaim
- claim3 -> booleanClaim

Reply:
Done!